startup/gui/menus.py

```diff
@@ -466,6 +466,7 @@ def __shaderNodeCreator( nodeName, shaderName ) :
 nodeMenu.append( "/VDB/Mesh To Level Set", GafferVDB.MeshToLevelSet, searchText = "MeshToLevelSet" )
 nodeMenu.append( "/VDB/Level Set Offset", GafferVDB.LevelSetOffset, searchText = "LevelSetOffset" )
 nodeMenu.append( "/VDB/Points Grid To Points", GafferVDB.PointsGridToPoints, searchText = "PointsGridToPoints" )
+nodeMenu.append( "/VDB/Sphere LevelSet", GafferVDB.SphereLevelSet, searchText="SphereLevelSet")
```

Comment:
"Sphere LevelSet" -> "Sphere Level Set". We seem to be treating it as two words in the other menu items at least.